htmd/parameterization/cli.py

```diff
@@ -563,6 +571,7 @@ def main_parameterize(arguments=None, progress=None):
     # Parse arguments
     parser = getArgumentParser()
     args = parser.parse_args(args=arguments)
+    args.queue = 'PlayQueue' if args.pm_token else args.queue
```

Comment:
is this really necessary?

Reply:
This is just for consistency and to make the argument logs to show the actual queue in use:
```
...
2019-09-02 18:19:24 - INFO -    optimize_dihedral: True
2019-09-02 18:19:24 - INFO -               outdir: /data/out
2019-09-02 18:19:24 - INFO -                queue: PlayQueue
2019-09-02 18:19:24 - INFO -              rtf_prm: None
2019-09-02 18:19:24 - INFO -                 seed: 20170920
...
```